gamestonk_terminal/stocks/sector_industry_analysis/sia_controller.py

```diff
@@ -38,13 +39,18 @@ class SectorIndustryAnalysisController(BaseController):
         "country",
         "mktcap",
         "exchange",
+        "period",
         "cps",
         "cpic",
         "cpis",
         "cpcs",
         "cpci",
         "sama",
         "metric",
+        "bs",
+        "cf",
+        "is",
+        "visualise",
```

Comment:
visualise is a long word, is there anything shorter we can use? Also @jmaslek will kill you for writing it the European way 😄